packages/lesswrong/server/migrations/20221207T030042.books_toc_title.ts

```diff
@@ -38,14 +37,10 @@ import PgCollection from "../../lib/sql/PgCollection";
 export const acceptsSchemaHash = "dc1ea5409f03e1b22c4c5835fd70e2a3";
 
 export const up = async ({db}: MigrationContext) => {
-  if (!Books.isPostgres()) {
-    throw new Error('Can only run this migration on postgres')
+  if (Books.isPostgres()) {
+    await addField(db, Books, "tocTitle");
+  } else {
```

Comment:
Slight preference for early return on if `!Books.isPostgres()` if there was any more than one line of code there, but there's only one line of code there, so w/e.